test/quantization/core/test_quantized_op.py

```diff
@@ -253,7 +253,29 @@ def test_qrelu(self, X):
                 }
             }
         ]
-        self._test_activation_function(X, 'relu', relu_test_configs)
+        devices = ["cpu", "cuda"] if TEST_CUDA else ["cpu"]
+        for device in devices:
+            # Only test the non-in-place version relu quantized cuda,
+            # will remove this when creating in-place version relu quantized cuda.
+            if device == 'cuda':
+                relu_test_configs = [
+                    {
+                        'quantized_fn': [
+                            torch.relu,
+                            torch.nn.functional.relu,
+                        ],
+                        'reference_fn': torch.nn.functional.relu
+                    },
+                ]
+            shapes = ((4,), (4, 4), (4, 4, 4), (4, 4, 4, 4))
+            dtypes = (torch.quint8, torch.qint8)
+            scales = (0.05, 0.1)
+            zero_points = (0, 5)
+            test_cases = itertools.product(shapes, dtypes, scales, zero_points)
+            for shape, dtype, scale, zero_point in test_cases:
+                X = torch.randn(*shape, device=device)
+                X = (X, (scale, zero_point, dtype))
+                self._test_activation_function(X, 'relu', relu_test_configs)
```

Comment:
Is there a reason why you are removing he hypothesis, and manually writing the tests? Although it is one of the plans, it should not be done within the PR that introduces a new feature.

Reply:
If we keep the he hypothesis, assuming we are doing tests on both 'cpu' and 'cuda', the first iteration, we need to cast a NumPy array to a 'cpu' Tensor, and the second time, we need to either send the tensor to 'cuda' or keep the original NumPy array and cast it to 'cuda'. I think both ways will make the code longer.